### PR TITLE
Update swagger generated code to MS68 release (removal of deployment template endpoints)

### PR DESCRIPTION
**Pull request by juliocamarero**
This PR updates the API with changes from ms-68 (still not removing the clusters API) but it does remove the old deployment templates endpoint and models.

The new endpoint:
- requires the region to be passed as a parameter instead of the path
- does not support formats (clusters vs deployments)

Rel: https://github.com/elastic/cloud/issues/93007

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,27 +1,33 @@
 cloud-sdk-go
-Copyright 2021 Elasticsearch B.V.
+Copyright 2021-2022 Elasticsearch B.V.
```
Comment (tobio) [inline NOTICE:2]: IANAL but wouldn't this just be `2022`, I'm guessing this file got automatically generated though.

File: pkg/models/deployment_upgrade_assistant_status_response.go [hunk 2]
```diff
@@ -31,32 +31,29 @@ import (
 	"github.com/go-openapi/validate"
 )
 
-// RemoteClusterInfo Information about the remote cluster for cross-cluster search.
```
Comment (tobio) [inline pkg/models/deployment_upgrade_assistant_status_response.go:34]: Git renames are sometimes brilliant.

**Review by tobio — APPROVED** [review]
LGTM, pretty clear replacement of the deprecated APIs. Build is failing on unused variables in `pkg/api/platformapi/configurationtemplateapi/get.go`.

File: NOTICE (hunk 1 above)
Comment (juliocamarero) [reply NOTICE:2]: Yup. Even if I change it manually to be just `2022`, it goes back to `2021-2022` when running `make swagger`

File: pkg/models/deployment_upgrade_assistant_status_response.go (hunk 2 above)
Comment (juliocamarero) [reply pkg/models/deployment_upgrade_assistant_status_response.go:34]: wow, that makes it really hard to review 🤷

**Comment by juliocamarero** [comment]
I've manually tested some of the commands agains QA (list, show, pull) and they all seemed to work, so merging this one. cc @elastic/cloud-delivery

**Review by rhass — APPROVED** [review]
LGTM

**Comment by jeredding** [comment]
@juliocamarero looks like commits need to be signed

**Pull request merged**